### PR TITLE
Added DeepSource in SAST list

### PR DESCRIPTION
**Pull request by saif-deepsource**
DeepSource helps you automatically find and fix issues in your code during code reviews, such as bug risks, anti-patterns, performance issues, and security flaws. It takes less than 5 minutes to set up with your Bitbucket, GitHub, or GitLab account. It works for Python, Go, Ruby, and JavaScript.

### CODE REVIEW

File: _data/tools.json [hunk 1]
```diff
@@ -962,6 +962,16 @@
       "note": null,
       "type": "SAST"
    },
+   {
+      "title": "DeepSource",
+      "url": "https://deepsource.io/",
+      "owner": null,
+      "license": "Commercial",
```
Comment (kingthorin) [inline _data/tools.json:969]: It's commercial but no-one owns it?

File: _data/tools.json [hunk 2]
```diff
@@ -962,6 +962,16 @@
...
+      "platforms": "SaaS or On-Premises",
+      "note": "DeepSource helps you automatically find and fix issues in your code during code reviews, such as bug risks, anti-patterns, performance issues, and security flaws. It takes less than 5 minutes to set up with your Bitbucket, GitHub, or GitLab account. It works for Python, Go, Ruby, and JavaScript.
+",
```
Comment (kingthorin) [inline _data/tools.json:972]: Please move the closing quote and comma to the previous line to address the JSON validity issue.

**Review by kingthorin — CHANGES_REQUESTED** [review]

File: _data/tools.json (hunk 1 above)
Comment (saif-deepsource) [reply _data/tools.json:969]: My bad, I'll add the company name.

File: _data/tools.json (hunk 2 above)
Comment (saif-deepsource) [reply _data/tools.json:972]: Sure, thanks for the suggestion.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**